### PR TITLE
create UI with blade

### PR DESCRIPTION
**Pull request by hoangnbvn**

### CODE REVIEW

**Comment by hoangnbvn** [comment]
ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -36,7 +38,9 @@ public function store(Request $request)
      */
     public function show(User $user)
     {
-        //
+        return view('user.show', [
+            'user' => $user,
+        ]);
```
Comment (ghost) [inline app/Http/Controllers/UserController.php]:
```suggestion
        return view('user.show', compact('user'));
```

File: resources/views/user/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <x-primary-button class="mt-4 mb-4" >
+                {{ __('Create New User') }}
+            </x-primary-button>
+            <table class="table w-full">
+                <thead>
+                    <tr>
+                        <th class="text-gray-900 dark:text-white" scope="col">#</th>
+                        <th class="text-gray-900 dark:text-white" scope="col">Name</th>
+                        <th class="text-gray-900 dark:text-white" scope="col">Username</th>
+                        <th class="text-gray-900 dark:text-white" scope="col">Action</th>
```
Comment (ghost) [inline resources/views/user/index.blade.php]: localization

File: resources/views/user/show.blade.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    {{ __("Create task") }}
+                </x-primary-button>
+            </x-nav-link>
+            <table class="table w-full">
+                <thead>
+                    <tr>
+                        <th class="text-gray-900 dark:text-white" scope="col">#</th>
+                        <th class="text-gray-900 dark:text-white" scope="col">Content</th>
+                        <th class="text-gray-900 dark:text-white" scope="col">Deadline</th>
+                        <th class="text-gray-900 dark:text-white" scope="col">Action</th>
```
Comment (ghost) [inline resources/views/user/show.blade.php]: localization

File: resources/views/user/show.blade.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                                {{ __('Delete task') }}
+                            </x-danger-button>
+                        </td>
+                    </tr>
+                    @endforeach
+                </tbody>
+            </table>
+        </div>
+    </div>
+</x-app-layout>
```
Comment (ghost) [inline resources/views/user/show.blade.php]: eof

**Comment by hoangnbvn** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**